### DISCUSSION TITLE
Have a recurring job running every 8 hours starting from 07:00 AM

### DISCUSSION THREAD

**Discussion by advapiIT** [quartznet/quartznet#956] [General] [answered]
Hello,
Till now I've used Quartz.Net to run jobs every x hours and it works great... how I've been asked to have a job running 3 times a day, at 07:AM and every next 8 hours. I was wondering if it's possible to have a recurring job starting at a specified time (maybe tomorrow it's changed to 09:00AM and every +4 hours,for example) or if it's best to have an array of cron expressions and set them for the same job (if possible)

Any suggestion?
Thanks

**Comment by lahma** [comment, marked as answer]
Cross-posting is a really bad practice.

https://stackoverflow.com/questions/63826497/running-quartz-net-for-a-specific-job-every-8-hours-starting-from-0700-am

**Comment by advapiIT** [comment]
sorry but I had no answer...
